infra/lib/lambdaBuilder/assetFunctions.ts

```diff
@@ -186,6 +189,14 @@ export function buildDownloadAssetFunction(
     assetStorageTable.grantReadData(downloadAssetFunction);
     suppressCdkNagErrorsByGrantReadWrite(scope);
 
+    const ssmParam = new ssm.StringParameter(scope, `/veerum/${stackName}/downloadAssetServiceRole/arn`, {
```

Comment:
why is this inside the buildDownloadAssetFunction? It's more logically to put this close to the downloadAssetServiceRole itself.

Reply:
Because downloadAssetServiceRole is auto generated when downloadAssetFunction created, to access to it name I need to be close to downloadAssetFunction
`downloadAssetFunction.role?.roleArn`